[2.14.0] iOS build error

Not building after update from 2.13.3 to 2.14.0

```
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/bardiamist/Library/Developer/Xcode/DerivedData/proj-gcfwctharjklzxamsqckslyxgkbd/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/A0Auth0.build/Objects-normal/arm64/A0Auth0.o /Users/bardiamist/Documents/proj/proj-mobile/node_modules/react-native-auth0/ios/A0Auth0.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'A0Auth0' from project 'Pods')
(1 failure)
```

<details>
  <summary>Screenshot</summary>
  
<img width="1078" alt="Screenshot 2022-10-10 at 16 58 07" src="https://user-images.githubusercontent.com/22082342/194841624-4a56c227-f8c7-4b8d-a787-af0842c05787.png">
</details>

--- a/A0Auth0.podspec
+++ b/A0Auth0.podspec
@@ -13,6 +13,7 @@ Pod::Spec.new do |s|
   s.source       = { :git => 'https://github.com/auth0/react-native-auth0.git', :tag => "v#{s.version}" }
 
   s.source_files = 'ios/**/*.{h,m,mm,swift}'
+  s.pod_target_xcconfig = { 'HEADER_SEARCH_PATHS' => "'${PODS_CONFIGURATION_BUILD_DIR}/#{s.name}/#{s.name}.framework/Headers'" }
   s.requires_arc = true
 
   s.dependency 'React-Core'

